### PR TITLE
Add port 443 to the connection example. 

### PR DESCRIPTION
**Pull request by davidrhodus**
Most people are trying to connect using port 9944 and not 443 per the nginx configuration file. I encountered the same issue and found similar discussions on Discord.

### CODE REVIEW

**Comment by davidrhodus** [comment]
@NukeManDan for your review 😀

**Comment by nuke-web3** [comment]
I am not the pro on integrations, nor do I have push abilities for this repo. But this looks like a good suggestion to add to me 😀

File: docs/maintain-wss.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ produce a "Not private" warning which you can skip by going to "Advanced" and th
 
 Open [Polkadot-JS UI](https://polkadot.js.org/apps) and click the logo in the top left to switch the
 node. Activate the "Development" toggle and input your node's address - either the domain or the IP
-address. Remember to prefix with `wss://`.
+address. Remember to prefix with `wss://example.com:443`.
```
Comment (Swader) [inline docs/maintain-wss.md]:
```suggestion
address. Remember to prefix with `wss://` and if you're using the 443 port, append `:443`, like so: `wss://example.com:443`.
```

**Review by Swader — CHANGES_REQUESTED** [review]
This assumes everyone is on 443, so still not good. I made a small change. Please merge, run prettier on it, and I'll merge the PR.

**Pull request merged**